packages/nextjs/src/server/index.ts

```diff
@@ -1,7 +1,15 @@
-export * from './clerkClient';
+/**
+ * Generic exports
+ */
+export { verifyToken, createClerkClient } from '@clerk/backend';
```

Comment:
After [this PR](https://github.com/clerk/javascript/pull/2365) is being merged this line will be the same as the public API of the ( eg `export * from @clerk/backend`)

Reply:
I feel its still easier for us to control the public API of the high level packages if we're explicit - we're not adding new APIs that often so it shouldn't be an issue. Happy to revise of course!